packages/api/src/schema/api-schema.yaml

```diff
@@ -1806,6 +1806,32 @@ components:
               leftAt:
                 type: integer
                 description: the time the participant left
+    usage-metric:
+      type: object
+      description: |
+        An individual metric about usage of a user.
+      required:
+        - UserID
+        - CreatorID
+        - DeliveryUsageMins
+        - TotalUsageMins
+        - StorageUsageMins
```

Comment:
I missed on the `livepeer-data` review that these fields did not have their JSON fields specified. We currently have `lowerCamel` on the public Studio API and `snake_case` for Catalyst APIs, so this would create yet another casing scheme which IMO gets pretty messy. It is not an absolute requirement here, especially not for this PR that is only a reference to the other one, but it would be good not to add yet another scheme to our APIs.

The fix can be done relatively simply as well, just by adding a `json` tag on the same line where you have the `bigquery` tags for the usage rows.